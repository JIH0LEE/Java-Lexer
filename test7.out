<CHARACTER,'s'>
<CHARACTER,'1'>
<CHARACTER,' '>
