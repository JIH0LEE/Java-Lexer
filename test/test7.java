char function_test(){
    if(3>2){
        int num=0;
    }
    return '1';
}