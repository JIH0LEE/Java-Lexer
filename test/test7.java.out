<1,VTYPE,char>
<1,ID,function_test>
<1,LPAREN,(>
<1,RPAREN,)>
<1,LBRACE,{>
<2,IF,if>
<2,LPAREN,(>
<2,NUM,3>
<2,COMP,>>
<2,NUM,2>
<2,RPAREN,)>
<2,LBRACE,{>
<3,VTYPE,int>
<3,ID,num>
<3,ASSIGN,=>
<3,NUM,0>
<3,SEMI,;>
<4,RBRACE,}>
<5,RETURN,return>
<5,CHARACTER,'1'>
<5,SEMI,;>
<6,RBRACE,}>
